nodes.adoc

```diff
@@ -0,0 +1,1163 @@
+----
+
+## Configuring cluster memory to meet container memory and risk requirements
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+## Estimating the number of pods your {product-title} nodes can hold
+
+[source,terminal]
+----
+$ oc create sa cluster-capacity-sa
+----
+
+
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user cluster-capacity-role \
+    system:serviceaccount:default:cluster-capacity-sa
+----
+
+
+[source,terminal]
+----
+$ oc create -f cluster-capacity-job.yaml
+----
+
+## Allowing containers to consume API objects
+
+[source,terminal]
+----
+$ oc create -f pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f volume-pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f volume-pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f secret.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f configmap.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f pod.yaml
+----
+
+## Using Init Containers to perform tasks before a pod is deployed
+
+[source,terminal]
+----
+$ oc create -f myapp.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f mydb.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f myservice.yaml
+----
+
+## Using port forwarding to access applications in a container
+
+[source,terminal]
+----
+$ oc port-forward <pod> [<local_port>:]<remote_port> [...[<local_port_n>:]<remote_port_n>]
+----
+
+
+[source,terminal]
+----
+$ oc port-forward <pod> [<local_port>:]<remote_port> [...[<local_port_n>:]<remote_port_n>]
+----
+
+
+[source,terminal]
+----
+$ oc port-forward <pod> 5000 6000
+----
+
+
+[source,terminal]
+----
+$ oc port-forward <pod> 8888:5000
+----
+
+
+[source,terminal]
+----
+$ oc port-forward <pod> :5000
+----
+
+
+[source,terminal]
+----
+$ oc port-forward <pod> 0:5000
+----
+
+## Mapping volumes using projected volumes
+
+[source,terminal]
+----
+$ oc create -f <secrets-filename>
+----
+
+
+[source,terminal]
+----
+$ oc create -f secret.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f <your_yaml_file>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f secret-pod.yaml
+----
+
+## Using sysctls in containers
+
+[source,terminal]
+----
+$ oc apply -f <file-name>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc edit machineconfigpool worker
+----
+
+
+[source,terminal]
+----
+$ oc apply -f set-sysctl-worker.yaml
+----
+
+## Using volumes to persist container data
+
+[source,terminal]
+----
+$ oc set volume <object_selection> <operation> <mandatory_parameters> <options>
+----
+
+
+[source,terminal]
+----
+$ oc set volume <object_type>/<name> [options]
+----
+
+
+[source,terminal]
+----
+$ oc set volume pod/p1
+----
+
+
+[source,terminal]
+----
+$ oc set volume dc --all --name=v1
```

Comment:
Requires a declarative alternative

Reply:
@siamaksade Why does this command require a declarative? The command only generates output that is specific to current DCs only.

```
$ oc set volume dc --all --name=data
 postgresql
  empty directory as data
    mounted at /var/lib/pgsql/data
 rails-postgresql-example
error: deploymentconfigs/rails-postgresql-example volume "data" not found
```

Reply:
Agreed. This one does not need any, provided we have displayed the declarative alternative in the previous commands.